async_test.go

```diff
@@ -124,7 +133,7 @@ func TestMultipleAsyncQueries(t *testing.T) {
 	})
 }
 
-func retrieveRows(rows *RowsExtended, ch chan string) {
```

Comment:
What is the reason to replace `RowsExtended` with `sql.Rows`?

Reply:
Because I changed function providing rows - from `dbt.mustQueryContext` to `db.QueryContext`. It returns `Rows` instead of `RowsExtended`. But we don't use features of `RowsExtended` so this is fine IMHO.